javascript/javascript1/week1/age.js

```diff
@@ -0,0 +1,69 @@
+let yearOfBirth = 1995;
+let yearFuture = 2027;
+let age = yearOfBirth - yearFuture;
+console.log("You will be " + age + "years old in " + yearFuture + ".");
```

Comment:
It is good practice to use `const` as much as possible. `Let` is mainly for loops and algorithms. This is relevant throughout the whole exercise.